prisma-fmt/tests/text_document_completion/scenarios/referential_actions_relation_mode_prisma/result.json

```diff
@@ -0,0 +1,25 @@
+{
+  "isIncomplete": false,
+  "items": [
+    {
+      "label": "Cascade",
+      "kind": 13,
+      "detail": "Delete the child records when the parent record is deleted."
+    },
+    {
+      "label": "Restrict",
+      "kind": 13,
+      "detail": "Prevent deleting a parent record as long as it is referenced."
+    },
+    {
+      "label": "NoAction",
+      "kind": 13,
+      "detail": "Prevent deleting a parent record as long as it is referenced."
+    },
+    {
+      "label": "SetNull",
+      "kind": 13,
+      "detail": "Set the referencing fields to NULL when the referenced record is deleted."
+    }
+  ]
+}
```

Comment:
Test highlighting the correct referential action completions for mongodb

Reply:
Just to complete the tests, maybe also add one for MySQL but with `relationMode=prisma`? 
(MongoDB is always set to `relationMode=prisma`, so might work a bit different under the hood.)